lib/node_modules/@stdlib/math/base/special/sincospi/examples/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+var linspace = require( '@stdlib/math/utils/linspace' );
+var sincospi = require( './../lib' );
+
+var x = linspace( 0.0, 2.0, 101 );
```

Comment:
Should match the example in the `README.md` file.

Reply:
Modified the readme to match this.